<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerMainWindowClass</class>
 <widget class="QMainWindow" name="VideoPlayerMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayerMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="video_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(225, 225, 225);
font-size: 30px;
</string>
    </property>
    <property name="text">
     <string>NO VIDEO</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>帧:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pos">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_total">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>731</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="open_btn">
       <property name="text">
        <string>打开视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&lt;&lt;慢放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>播放/暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>快进&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_mag">
       <property name="text">
        <string> 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frame_handle_btn">
       <property name="text">
        <string>文件处理</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VideoPlayerMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
